CONTRIBUTING

```diff
@@ -0,0 +1,28 @@
+# Contributing
+
+Hey, thanks for your interest in contributing to the NYU CS Wiki! Please review this document **before making a pull request**.
+
+## Guidelines and Style
+
```

Comment:
I would add articles are written in markdown and link to a markdown tutorial. Articles may also have some front matter like post title, sidebar position, tags, etc. Could explain a bit about those/say look at existing articles as examples.